### PR TITLE
Setup share features

### PR DESCRIPTION
**Pull request by jabarros**

### CODE REVIEW

**Comment by davivel** [comment]
@DeepDiver1975 , we need to include @jabarros in cla-bot-android; he is back with us from today.

File: res/values/setup.xml [hunk 1]
```diff
@@ -24,7 +24,8 @@
     
     <!-- Flags to enable/disable some features -->
     <string name = "send_files_to_other_apps">on</string>
-    <string name = "share_feature">on</string>
+    <string name = "share_via_link_feature">on</string>
+    <string name = "share_with_users_feature">on</string>
```
Comment (malkomich) [inline res/values/setup.xml]: Flag values could be treated directly as boolean instead of parsing the string to get a boolean value.

**Comment by AndyScherzinger** [comment]
Since this code change is already reworking the share UI, it could also implement #1623. Just a thought :)

**Comment by malkomich** [comment]
#1623 requires a small change in the order of the layout elements, so I agree with this.

**Comment by malkomich** [comment]
It looks good for me :+1:

**Comment by jabarros** [comment]
@AndyScherzinger I have included the changes in sorting of »Share link« settings (#1623) on this [commit](https://github.com/owncloud/android/pull/1631/commits/0834aa3a50239f9bcd4629f0909143b56bb7628c).

**Comment by davivel** [comment]
You all cheaters sneaking other issues... 

@DeepDiver1975 , really needing that @jabarros is added to cla-bot-android. Thanks.

**Comment by AndyScherzinger** [comment]
@jabarros awesome! :+1:

**Comment by davivel** [comment]
Validated. Please, @jabarros , rebase ```setup_share_features``` on ```master```

**Comment by davivel** [comment]
Everything ready.

**Pull request merged**